lib-utilities/services/rpcService.go

```diff
@@ -78,3 +85,124 @@ func InitializeService(serviceName string) error {
 	return nil
 
 }
+
+// gRPC implementation starts here. TODO: remove this comment after the removal of go micro implementation.
+
+type serviceType int
+
+const (
+	serverService serviceType = iota
+	clientService
+)
+
+var (
+	// GRPCClientTransportCreds is for establishing the tls secured gRPC client communication
+	GRPCClientTransportCreds credentials.TransportCredentials
+	// GRPCServer is for bringing up gRPC micro services
+	GRPCServer *grpc.Server
+)
+
+// InitializeGRPC initializes the gRPC client transport and server
+// Function returns error at the failure of server or client transport creation
+func InitializeGRPC() error {
```

Comment:
Are we not implementing Service interface like go micro ?

Reply:
@Bharath-KKB Addressed.